composer.json

```diff
@@ -11,13 +11,13 @@
         }
     ],
     "require": {
-        "php": "^7.0",
+        "php": "^7.1|^8.0",
```

Comment:
Couldn't we stay with `^7.0` as the minimum (`^7.0|^8.0`) ?

Reply:
Yes, referring to the other comment, this is only to satisfy the requirements of more modern PHPUnit versions, which require the `:void` return declaration to match. If you decide to leave the PHPUnit version pinned to v6, then 7.0 will work fine.